scripts/dev-install.sh

```diff
@@ -16,7 +16,7 @@ install_all_in_edit_mode () {
     #  1. without deps -- install odc-* but none of their dependents
     #  2. with deps -- now that all `odc-*` are here, reinstall getting all other deps
     python3 -m pip install --no-deps -r "${reqs}"
-    python3 -m pip install -r "${reqs}"
+    python3 -m pip install --extra-index-url https://packages.dea.ga.gov.au/ -r "${reqs}"
```

Comment:
You should not need that `--extra-index-url`.

Reply:
Without it it can't find hdstats.